content/influxdb3/core/install.md

```diff
@@ -91,13 +91,13 @@ source ~/.zshrc
 
 <!-------------------------------- BEGIN LINUX -------------------------------->
 
-- [{{< product-name >}} • Linux (x86) • GNU](https://download.influxdata.com/influxdb/snapshots/influxdb3-{{< product-key >}}_x86_64-unknown-linux-gnu.tar.gz)
+- [{{< product-name >}} • Linux (AMD64, x86_64) • GNU](https://dl.influxdata.com/influxdb/releases/influxdb3-{{< product-key >}}-3.0.1_linux_amd64.tar.gz)
```

Comment:
We should be able to replace `3.0.1` with `{{% latest-patch %}}`, but I made need to refactor the shortcode a bit and update `data/products.yml` to get it to work with v3 products. I'll look into it.

Reply:
> We should be able to replace `3.0.1` with `{{% latest-patch %}}`, but I may need to refactor the shortcode a bit and update `data/products.yml` to get it to work with v3 products. I'll look into it.

I was thinking the same thing, but assumed it's not urgent. I can give it a shot if you have bigger fish.